netlify.toml

```diff
@@ -32,7 +32,7 @@
   REACT_APP_BRIDGE_API_URI="https://toucan-bridge-staging.herokuapp.com"
   REACT_APP_BRIDGE_CREDIT_CLASS_ID = "C56"
   REACT_APP_BRIDGE="true"
-  REACT_APP_SKIPPED_CLASS_ID="C01"
+  REACT_APP_SKIPPED_CLASS_ID=""
```

Comment:
why changing this?
although I guess this doesn't really matter too much in dev/staging

Reply:
Because all our sell orders are on `C01` on redwood, I changed it so we can use them again for testing.